version

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Version script for Marathon.
+#
+# Change MINOR to x+1 and BRANCH_POINT to commit hash of common ancestor of master and releases/1.x
+# after a releases/1.x was cut
+MAJOR=1
```

Comment:
What is the advantage of getting the version from `version` file instead of getting it from `git`?

Reply:
`git` is not so flexible at least I did not find it too be so. Also, major and minor version have to be persisted somewhere. Matthias, Ken and I agreed that it should not be a tag since tags should be reserved for community releases.